terraform/deployments/govuk-publishing-platform/content_schemas.tf

```diff
@@ -9,9 +22,53 @@ resource "aws_s3_bucket" "content_schemas" {
   force_destroy = contains(["staging", "production"], var.govuk_environment) ? false : true
 
   tags = {
-    name            = "govuk-${var.govuk_environment}-${local.workspace}-content-schemas"
+    name            = local.content_schemas.s3_bucket
     aws_environment = var.govuk_environment
   }
 }
 
-# TODO: ECS must be able to cp objects in this bucket.
+module "content_schemas_container_definition" {
+  source              = "../../modules/container-definition"
+  image               = "430354129336.dkr.ecr.eu-west-1.amazonaws.com/govuk/ecs-cli:latest"
+  aws_region          = data.aws_region.current.name
+  command             = ["/bin/sh", "-c", "aws s3 cp s3://${local.content_schemas.s3_bucket} /govuk-content-schemas --recursive"]
```

Comment:
Same comment as here regarding the content schemas path.

Reply:
I've changed this to use `$GOVUK_CONTENT_SCHEMAS_PATH`.